src/@next/Button/ButtonStyle.ts

```diff
@@ -46,10 +47,24 @@ min-width: 67px;
   }
 `;
 
+const verySlimButtonSizeStyle = `
+  padding: 5px ${space2};
+
+  @media (max-width: ${Breakpoints.large}) {
+    padding: 0px 0px;
+  }
+
+  p {
+    margin-block-start: 0px;
+    margin-block-end: 0px;
+  }
```

Comment:
I encountered some weird vertical spaces affecting Typography component inside button, something like this:
![image](https://github.com/glints-dev/glints-aries/assets/67429929/3003ae5f-a0fe-4e8e-b687-0424d4fdac4d)

It says 'user agent stylesheet', when I researched it seems to be some default stylings by the browser, so I just overwrite those